python/az/aro/azext_aro/_format.py

```diff
@@ -19,5 +19,5 @@ def aro_show_table_format(result):
         Location=result['location'],
         ProvisioningState=result['provisioningState'],
         WorkerCount=result['workerProfiles'][0]['count'],
-        ConsoleURL=result['consoleUrl'],
+        URL=result['clusterProfile']['url'],
```

Comment:
1- typo s/clusterProfile/consoleProfile/
2- maybe continuing to say `ConsoleURL=` is a good idea? (We say WorkerCount= instead of Count= above?)

Reply:
1st changed. 
2. We do say `URL        string     `json:"url,omitempty"`` and we already created a dedicated structure for Console, so consoleURL sounds redundant.